.github/workflows/integration-test.yml

```diff
@@ -108,6 +107,8 @@ jobs:
           - test: 'nginx'
           - test: 'prometheus'
           - test: 'redis'
+          - test: 'splunk'
```

Comment:
Does it make sense to similarly name this one `splunk-8`? If we can't merge them into a single `splunk` test and want to verify compatibility across versions it may be useful.

Reply:
Yeah, I'd suggest refactoring this a bit to have the matrix be:

```
- test: splunk
  env:
    SPLUNK_VERSION: 7.3.9
- test: splunk
```

and have the below export the environment variables. This would let you continue to leverage the existing scaffolding for integration tests rather than needing to add a make target just for the Splunk 7 tests.

Reply:
Ah good clever solution 👍

Reply:
Updated in 242de25